lib/application/pages/execution/execution_terminal.dart

```diff
@@ -106,25 +106,27 @@ class ExecutionTerminal extends HookWidget {
       child: Text(actionText.toUpperCase()),
     );
 
-    return Container(
-      clipBehavior: Clip.hardEdge,
-      decoration: BoxDecoration(
-        color: Colors.transparent,
-        borderRadius: dimens.executionsTerminalBorderRadius,
-        border: Border.all(
-          color: borderColor,
-          width: dimens.executionsTerminalBorderWidth,
+    return SafeArea(
```

Comment:
I don't think this belongs to this particular widget. Instead, should be used by the page that uses this widget, because we are not sure where it would be used in the future. I would say somewhere [around here](https://github.com/olmps/memo/blob/main/lib/application/pages/execution/collection_execution_page.dart#L32).